javatests/arcs/core/data/BUILD

```diff
@@ -15,6 +15,7 @@ arcs_kt_jvm_test_suite(
         "//java/arcs/core/data",
         "//java/arcs/core/data:recipe_android_proto",
         "//java/arcs/core/testutil",
+        "//java/repoutils",
```

Comment:
Is there a reason you opted for //java/repoutils instead of //java/arcs/repoutils?

Reply:
I felt it is an artifact of the repo structure and not arcs itself and it should not even be in arcs package.

Reply:
It kind of goes against the whole Java reverse-domain package naming idea. Arguably our top-level "arcs" package does too (should probably be dev.arcs on account of us owning the arcs.dev domain name), but that's a problem for another day... 

We don't want naming conflicts with other libraries, and putting everything in our own "domain" is the correct way to ensure that.

Plus, here we are specifically talking about utils for dealing with the arcs repo, so arcs.repoutils as a package name actually makes a lot of sense.

Reply:
Ok, I'm buying this, updating :)